src/gui/plugins/component_inspector/Vector3d.qml

```diff
@@ -74,6 +78,32 @@ Rectangle {
       }
     }
   }
+  Component {
+      id: plotIcon
+      Image {
+          property string componentInfo: ""
+          source: "plottable_icon.svg"
+          anchors.top: parent.top
+          anchors.left: parent.left
+
+          Drag.mimeData: { "text/plain" : (model === null) ? "" :
+          "Component," + model.entity + "," + model.typeId + "," + model.dataType + "," + componentInfo}
```

Comment:
The `typeId` is convenient to uniquely identify each component type, but it isn't very user friendly on the UI. For example, it's hard for a user to understand that the pose x, y and z are being plotted here:

![model_plot](https://user-images.githubusercontent.com/5751272/91924965-e763b480-ec88-11ea-854a-e1138c204ac9.png)

Would it be possible to use `typeName` when displaying component information? The internal bookkeeping can remain using `typeId`.

Reply:
ok i made the modification for that PR
but the main code of that will be in the PlottingInterface code so i will update the other PR